charts/artifact-hub/values.yaml

```diff
@@ -372,6 +388,10 @@ trivy:
     #   requests:
     #     cpu: 100m
     #     memory: 128Mi
+    # Optionally specify extra list of additional labels for the hub deployment
```

Comment:
Can you replace `hub` with `trivy`? There is another occurrence a couple of lines below.